<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="minimumSize">
       <size>
        <width>383</width>
        <height>568</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="aLayout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>566</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="resultLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>直方图均衡化</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>傅里叶变化(未完成)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>缩放图片</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>旋转图片</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>灰度窗调整</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>图像增强</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>灰度反转</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>图像左右翻转</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>383</width>
        <height>568</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>382</width>
         <height>626</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="originImage">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label1">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>原始图片</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="grayImage">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>灰度图片</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="remarkLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionread_binary"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_to"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionParameters"/>
    <addaction name="actionExport_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>读取图片</string>
   </property>
  </action>
  <action name="actionSave_to">
   <property name="text">
    <string>灰度图片另存为</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="text">
    <string>功能参数设置</string>
   </property>
  </action>
  <action name="actionExport_Settings">
   <property name="text">
    <string>导出图片设置</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>结果图片另存为</string>
   </property>
  </action>
  <action name="actionread_binary">
   <property name="text">
    <string>读取二进制文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
